### ISSUE TITLE
Graphic menu suggestions

### ISSUE THREAD

**Issue by Martin56700** [numworks/epsilon#599]
Hello,

here suggestions for graphic menu :
![maquettes-menu-functions](https://user-images.githubusercontent.com/40736589/42180402-68f563d0-7e37-11e8-9ff3-8f8b6f0662b5.png)

**Comment by matheod** [comment]
I find the empty sections confusing.

**Comment by Martin56700** [comment]
Really. In the first time, why not ! And now, when you look a new time, are you the same feeling ?

**Comment by boricj** [comment]
I've tried to make a clock work (#311), but basically the only two time sources available are either completely unusable for precise time-keeping (the LSI) or would drain the batteries flat in a couple of weeks (the HSE). The LSI could be used as a timer active only while in exam mode or a HSE clock could be made opt-in, but to be honest if you want a NumWorks calculator with an integrated clock you should ask the NumWorks team to add a LSE oscillator whenever they'll release a new model.

While I'm talking about the clock, remember that the calculator has a 320x240 screen resolution and two fixed-size fonts. You can't fit both a clock and a date vertically on the status bar without doubling its height and screen estate is at a premium here.

**Comment by Martin56700** [comment]
I agree with you on all points. There isn't cristal oscillator in the cpu ?
[https://www.reddit.com/r/numworks/comments/8es5g2/real_time_clock/](url)

But, what do you want to say by : "screen estate is at a premium here."

**Comment by boricj** [comment]
The STM32F4 MCU has two internal RC oscillators (LSI/HSI). The motherboard has an external high frequency crystal oscillator fitted (HSE) for USB communication, but doesn't have the equivalent low-speed one (LSE). Check out the clock source characteristics on the MCU datasheet for more details.

For the screen, space is precious on the device. There's simply not a lot of pixels to go around, especially when compared to computer monitors. It's easy to design stuff on the computer that won't work on the device: one example is that the two monospaced fonts on the device take a lot more room than the TrueType fonts you've used, so the date as you formatted it would most likely eat into the exam mode icon. Vectorized graphics, zoom-ins and linear interpolation on a high DPI monitor can also trick you into thinking something is working whereas in fact it'd be a smudge on the low-res, poorly contrasted and washed-out colors NumWorks calculator screen (my ported NumWorks firmware looks a LOT better on my HP Prime).

I just spotted another impossible thing: you added a battery percent indicator, but the calculator has a divider bridge coupled with battery voltage measurement: cheap but extremely imprecise. You need a real battery controller IC to keep track of battery charge, which the NumWorks calculator doesn't have.

**Comment by RedGl0w** [comment]
Note : partially duplicate of https://github.com/numworks/epsilon/issues/139
